### ISSUE TITLE
Passing probability into CrossEntropy 

### ISSUE THREAD

**Issue by Tma2333** [yiyixuxu/polyloss-pytorch#5]
`torch.nn.CrossEntropyLoss` expect you to pass raw, unnormalized logit, however, either logit or probability is passing into cross entropy loss. If the probability is used will lead to unintended result.
